Sim/SimpleFtl/SimpleFtl.cpp

```diff
@@ -58,29 +62,29 @@ void SimpleFtl::ProcessEvent()
     _EventQueue->pop(event);
     switch (event.EventType)
     {
-        case Event::Type::CustomProtocolCommand:
-        {
-            OnNewCustomProtocolCommand(event.EventParams.CustomProtocolCommand);
-        } break;
+    case Event::Type::CustomProtocolCommand:

```

Comment:
Is this indentation style intentional?

Reply:
Just updated the indentation for switch case.